src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -17,178 +17,165 @@ import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import StarBorderIcon from '@material-ui/icons/StarBorder';
 import ClearIcon from '@material-ui/icons/Clear';
 import user from '../../../../../../../../services/user';
-import '../../../../../../apartmentApp.css';
-import '../../../../../../../PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
 
 // Based off src/views/PeopleSearch/components/PeopleSearchResult
-// but using this.props.profile of type StudentProfileInfo
+// but using props.profile of type StudentProfileInfo
 // rather than using this.props.Person of type PeopleSearchResult
-export default class ApplicantListItem extends Component {
-  constructor(props) {
-    super(props);
+const ApplicantListItem = (props) => {
```

Comment:
Yes, I noticed that going on in code such as 'Wellness' and I suspected that was what it meant. I figured i'd go back after and explore that, since I just needed to check the parent component to find out which components I had previously planned to use.

Reply:
It was helpful for pointing out which props were being passed but not being used (likely due to losing track of things that were added and removed over time with different features)